src/Discord.Net.Interactions/InteractionService.cs

```diff
@@ -53,11 +53,18 @@ public class InteractionService : IDisposable
         public event Func<IAutocompleteHandler, IInteractionContext, IResult, Task> AutocompleteHandlerExecuted { add { _autocompleteHandlerExecutedEvent.Add(value); } remove { _autocompleteHandlerExecutedEvent.Remove(value); } }
         internal readonly AsyncEvent<Func<IAutocompleteHandler, IInteractionContext, IResult, Task>> _autocompleteHandlerExecutedEvent = new();
 
+        /// <summary>
+        ///     Occurs when a Modal command is executed.
```

Comment:
> Modal 

Are we going to have some consistency around the naming? (i.e. `Modal` vs `modal` when used within docs)